DataOnion/AuthExtensions.cs

```diff
@@ -75,13 +86,31 @@ public IFluentAuthStategyOnion ConfigureRedis(string connectionString)
             return this;
         }
 
-        public IFluentRedisOnion AddCustomAuthStrategy<T>(IAuthServiceStrategy<T> authStrategy)
+        public IFluentAuthOnion AddCustomAuthStrategy<T>(IAuthServiceStrategy<T> authStrategy)
         {
             _serviceCollection.AddScoped(typeof(IAuthService<>), typeof(AuthService<>));
             _serviceCollection.AddSingleton<IAuthServiceStrategy<T>>(authStrategy);
 
             return this;
         }
+
+        public IFluentAuthOnion ConfigureTwoFactorAuth(
+            string envPrefix,
```

Comment:
This shouldn't be necessary as an argument here; it's already a property `_environmentPrefix`, since it was passed in from `AddAuthOnion`.

Reply:
Thanks for spotting this, this has been simplified